abilities/execution/eb814e03-811a-467a-bc6d-dcd453750fa2.yml

```diff
@@ -0,0 +1,15 @@
+---
+
+- id: eb814e03-811a-467a-bc6d-dcd453750fa2
+  name: Install PSTools
+  description: Install PSTools by unzipping the file
```

Comment:
let's remove ac22dcfd-33aa-42cd-89ef-cfd3aea78933 as part of this. if it is used inside an adversary, remove it from the adversary.